### DISCUSSION TITLE
Need suggestions - Running etcd as realtime process

### DISCUSSION THREAD

**Discussion by rajanarayanaa** [etcd-io/etcd#17474] [Q&A]
Is it advisable to run etcd as a realtime process using `chrt`?
Example `chrt -p 60 <pid of etcd process>` This is primarily to ensure etcd get enough CPU cycles on a system when it's loaded.
Could someone please advise me here?

**Comment by jmhbnz** [comment]
Hey @rajanarayanaa - Thanks for your question. etcd is generally more sensitive to disk latencies rather than CPU contention from my experience.

When etcd is running on a system with competing workloads our tuning guidelines recommend the use of `ionice` to set disk priority, refer: https://etcd.io/docs/v3.5/tuning/#disk.

We don't currently have any written tuning documentation for `etcd` with `chrt` so if you do experiment with this please let us know how it goes.
